ci-operator/config/codeready-toolchain/workload-analyzer/codeready-toolchain-workload-analyzer-master.yaml

```
@@ -0,0 +1,59 @@
+base_images:
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.22-openshift-4.17
+images:
+- dockerfile_path: test/e2e/alertmanager-mock/build/Dockerfile
+  from: base
+  to: alertmanager-mock
+- dockerfile_path: build/Dockerfile
+  from: base
+  to: workload-analyzer
```

I see that it's still failing, I'm wondering if it's not missing the inputs field
```suggestion
- dockerfile_path: build/Dockerfile
  from: base
  inputs:
    bin:
      paths:
      - destination_dir: .
        source_path: /go/src/github.com/codeready-toolchain/workload-analyzer/build/_output/bin/workload-analyzer
  to: workload-analyzer
```
if it still doesn't work, then just add a new dockerfile and build the WA image in there